bip-update-process.md

```diff
@@ -0,0 +1,646 @@
+BIP’s content should only be adjusted in minor details, e.g. to improve language, clarify ambiguities, backfill
+omissions in the specification, add test vectors for edge cases, or address other issues discovered as the BIP is
+adopted.
+
+A Proposed BIP can only move to Active or Abandoned. Any necessary changes to the specification should be
+minimal and interfere as little as possible with ongoing adoption. If a Proposed BIP is found to need
+substantial functional changes, it may be preferable to move it to Abandoned[^new-BIP], and to start a new BIP with the changes
+instead. Otherwise, it could cause confusion as to what being compliant with the BIP means.
+
+#### Active
+
+A settled[^settled] BIP may be advanced to Active upon request by
+any community member with evidence that the idea described in the BIP is in active use. Such evidence includes
+for example: two or more projects having deployed support for the BIP in mainnet software releases, a soft fork
+proposal’s activation criteria having been met on the network, or rough consensus for the BIP having been demonstrated.
+
+At that point, the BIP should be considered final and any breaking changes to the BIP should be proposed as a new
+separate BIP.[^new-BIP]
+
+##### Process BIPs
+
+A Process BIP may change status from Proposed to Active when it achieves rough consensus on the mailing list. Such a
+proposal is said to have rough consensus if it has been open to discussion on the development mailing list for at least
+one month, and no person maintains any unaddressed substantiated objections to it. Addressed or obstructive objections
+may be ignored/overruled by general agreement that they have been sufficiently addressed, but clear reasoning must be
+given in such circumstances. Active Process BIPs may be modified indefinitely as long as a proposed modification
+has rough consensus per the same criteria.[^living-documents]
+
+##### Revisions
+
+To help implementers understand updates to a BIP, any changes after it has reached Proposed or Active are
+tracked with version, date, and description in a Changelog section. The version number is inspired by semantic
+versioning (MAJOR.MINOR.PATCH). The MAJOR version is incremented if changes to the BIP’s Specification are introduced
+that are incompatible with prior versions (which should be an exception after a BIP is Proposed, and not happen to a
+BIP that is Active). The MINOR version is incremented whenever the specification of the BIP
+is changed or extended in a backward-compatible way. The PATCH version is incremented for other changes to the BIP that
+are noteworthy (bug fixes, test vectors, important clarifications, etc.). The version should start at 1.0.0 with the
+date the BIP was promoted to Proposed. An example of a Changelog section can be seen in
+[BIP 352](https://github.com/bitcoin/bips/blob/master/bip-0352.mediawiki). After a BIP receives a Changelog, the
```

Comment:
Ha, except that the section is called "Change Log" and not "Changelog". I prefer the latter. It's really a micro nit, but if we adopt the latter, we may want to change this BIP352 just for reference.

Reply:
Tend to agree, though for instance https://www.gnu.org/prep/standards/html_node/Style-of-Change-Logs.html#Style-of-Change-Logs writes them as "change log"...it would be good to settle on one of the two.

I plan to make the existing ones consistent, e.g. per https://keepachangelog.com/en/1.1.0/.

Reply:
I think it would make sense to make the Changelog sections consistent should this BIP be adopted, but the adherence of BIP 352 to this BIP before this BIP is adopted is a bit out of scope ^^